internal/pubsub/gcppubsub/gcppubsub.go

```diff
@@ -51,18 +51,18 @@ func OpenTopic(ctx context.Context, client *raw.PublisherClient, proj gcp.Projec
 // openTopic returns the driver for OpenTopic. This function exists so the test
 // harness can get the driver interface implementation if it needs to.
 func openTopic(ctx context.Context, client *raw.PublisherClient, proj gcp.ProjectID, topicName string) (driver.Topic, error) {
+	path := fmt.Sprintf("projects/%s/topics/%s", proj, topicName)
 	client, err := raw.NewPublisherClient(ctx)
 	if err != nil {
 		return nil, fmt.Errorf("gcppubsub: creating publisher client: %v", err)
 	}
-	ok, err := topicExists(ctx, client, topicName)
+	ok, err := topicExists(ctx, client, path)
 	if err != nil {
 		return nil, fmt.Errorf("gcppubsub: checking existence of topic: %v", err)
 	}
 	if !ok {
 		return nil, fmt.Errorf("gcppubsub: topic named %q does not exist", topicName)
 	}
-	path := fmt.Sprintf("projects/%s/topics/%s", proj, topicName)
 	return &topic{path, client}, nil
 }
 
```

Comment:
What is "_deleted-topic_"?

Reply:
It's a special topic name from the GCP PubSub API. The original code is here:

https://github.com/GoogleCloudPlatform/google-cloud-go/blob/977bdf6a60d16cd466ccbfe6c20bfc20ddf923ba/pubsub/topic.go#L276

Subscriptions to deleted topics get their topic name set to this value. 
https://github.com/GoogleCloudPlatform/google-cloud-go/blob/977bdf6a60d16cd466ccbfe6c20bfc20ddf923ba/pubsub/apiv1/publisher_client.go#L297

Reply:
Please add a comment explaining this and referencing the documentation.

Reply:
Done.